### PR TITLE
Fix layout of table views vs. nav bar.

### PR DESCRIPTION
**Pull request by charlesmchen**
PTAL @michaelkirk 

### Before (first row hidden by navigation bar)

img_6675

### After

img_6676

### CODE REVIEW

File: Signal/src/ViewControllers/ConversationView/Cells/OWSMessageCell.m [hunk 1]
```diff
@@ -257,13 +257,13 @@ - (UIFont *)textMessageFont
         case 0:
             break;
         case 1:
-            return [UIFont ows_regularFontWithSize:basePointSize * 2.f];
+            return [UIFont ows_regularFontWithSize:basePointSize + 18.f];
```
Comment (michaelkirk) [inline Signal/src/ViewControllers/ConversationView/Cells/OWSMessageCell.m]: Was this intentional? It seems unrelated.

Comment (charlesmchen) [reply Signal/src/ViewControllers/ConversationView/Cells/OWSMessageCell.m]: Removed!
